src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs

```diff
@@ -2,20 +2,21 @@ namespace OrganisationRegistry.ElasticSearch.Projections.Organisations
 {
```

Comment:
this is quite a PITA: 
![image](https://user-images.githubusercontent.com/3797702/157195299-8f4f1c67-a68b-47ab-b1d3-b5dce68a67d2.png)

Reply:
I'll look into it